### ISSUE TITLE
conflict with Booking Calendar plugin

### ISSUE THREAD

**Issue by tommyg13** [google/site-kit-wp#392]
## Bug Description
Hello i noticed that Site Kit by Google is not working with Booking Calendar plugin.  Site kit not showing on admin side menu and when i access directly /wp-admin/admin.php?page=googlesitekit-splash i get the following errors on console
Invariant Violation: Minified React error #200;
Invariant Violation: Minified React error #31;

When i deactivate Booking Calendar plugin (https://el.wordpress.org/plugins/booking/) everything works well

 - PHP Version:  7.3
 - Browser [e.g. chrome, safari] chrome, mozilla
 - Plugin Version [e.g. 22] Version 1.0.0-beta.1.0.3

![IKjiZaUqf9](https://user-images.githubusercontent.com/41326532/62882098-111e0680-bd29-11e9-81ab-fe441582bc23.gif)

Additional Context:
- Site Kit tab becomes active again once Booking plugin is deactivated
- Access Site Kit via URL (https://example.com/wp-admin/admin.php?page=googlesitekit-splash) results in a blank page (see below) with the following console error:

```
react-dom.min.js?ver=16.8.4:117 Invariant Violation: Minified React error #200; visit https://reactjs.org/docs/error-decoder.html?invariant=200 for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at pb (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:12:454)
    at n (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:13:221)
    at ch (https://example.iecomwp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:164:273)
    at t.value (https://example.com/wp-content/plugins/google-site-kit/dist/assets/js/commons.js?ver=1.0.0-beta.1.0.3:1:82943)
    at ce (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:98:136)
    at qg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:97:440)
    at hi (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:104:285)
    at Qg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
Bg @ react-dom.min.js?ver=16.8.4:117
react-dom.min.js?ver=16.8.4:117 Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Invariant%20Violation%3A%20Minified%20React%20error%20%23200%3B%20visit%20https%3A%2F%2Freactjs.org%2Fdocs%2Ferror-decoder.html%3Finvariant%3D200%20for%20the%20full%20message%20or%20use%20the%20non-minified%20dev%20environment%20for%20full%20errors%20and%20additional%20helpful%20warnings.%20&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at pb (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:12:454)
    at n (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:13:221)
    at Ac (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:70:369)
    at https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:80:190
    at N (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:90:3)
    at hi (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:105:408)
    at Qg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)
    at Z (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:156:492)
Bg @ react-dom.min.js?ver=16.8.4:117
react-dom.min.js?ver=16.8.4:12 Uncaught Invariant Violation: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Invariant%20Violation%3A%20Minified%20React%20error%20%23200%3B%20visit%20https%3A%2F%2Freactjs.org%2Fdocs%2Ferror-decoder.html%3Finvariant%3D200%20for%20the%20full%20message%20or%20use%20the%20non-minified%20dev%20environment%20for%20full%20errors%20and%20additional%20helpful%20warnings.%20&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
    at pb (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:12:454)
    at n (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:13:221)
    at Ac (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:70:369)
    at https://example.ie/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:80:190
    at N (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:90:3)
    at hi (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:105:408)
    at Qg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:144:217)
    at Rg (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:145:76)
    at Sc (https://example.com/wp-includes/js/dist/vendor/react-dom.min.js?ver=16.8.4:158:109)

```
![Screen Shot 2019-08-12 at 5 48 14 PM](https://user-images.githubusercontent.com/41326532/62882240-60fccd80-bd29-11e9-8674-cf02d3e06c0a.png)

XHR Response:
![Screen Shot 2019-08-12 at 5 51 52 PM](https://user-images.githubusercontent.com/41326532/62882489-e7191400-bd29-11e9-952e-44e5315fb51b.png)

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* 

## Implementation Brief

* 

## Changelog entry

*

**Comment by jamesozzie** [comment]
@tommyg13 Most likely there is an Google API PHP conflict which is the cause of your error.  (Similar to what you see in #246). Are you connecting the plugins calendar to your own Google calendar?

**Comment by tommyg13** [comment]
hello no i did not connect calendar with google calendar

**Comment by jamesozzie** [comment]
@tommyg13 There is an open bug for this issue now. I was able to replicate the same issue from my side. I will keep this support ticket open while it's being worked on.

**Comment by felixarntz** [comment]
@tommyg13 The issue was fixed as part of #401. It is a generic problem in WordPress core that in unfortunate circumstances different plugins' admin menu items may override each other. We implemented a workaround in #401 to make sure that doesn't happen.

**Tomorrow** version `1.0.0-beta.1.0.4` will be released, which will include this fix. Please update and retest afterwards. I'll close this issue, but don't hesitate reopening if it is still not solved after the update.
